region/lowernorfair/east/Fast Pillars Setup Room.json

```diff
@@ -1201,6 +1254,101 @@
+          ]},
+          {"and": [
+            "h_heatProof",
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"or": [
+                  "ScrewAttack",
+                  "canDodgeWhileShooting"
+                ]},
+                {"or": [
+                  {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+                  "canBeLucky"
+                ]},
+                {"or": [
+                  {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"]], "explicitWeapons": ["ScrewAttack", "Charge+Plasma", "Charge+Ice+Wave+Spazer", "Super"]}},
+                  {"and": [
+                    {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"]], "explicitWeapons": ["Charge"]}},
+                    "canBePatient"
+                  ]}
+                ]},
+                "h_usePowerBomb"
+              ]},
+              {"and": [
+                "canBeLucky",
+                {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"]], "explicitWeapons": ["Missile"]}},
+                "h_usePowerBomb"
+              ]},
+              {"and": [
+                {"enemyKill": {"enemies": [["Yellow Space Pirate (standing)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"]], "explicitWeapons": ["PowerBomb"]}},
+                "canTrickyDodgeEnemies"
+              ]}
+            ]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
+          ]}
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 12, "gentleDownTiles": 2, "gentleUpTiles": 2, "openEnd": 0}},
+          {"and": [
+            {"or": [
+              {"doorUnlockedAtNode": 2},
+              {"doorUnlockedAtNode": 3}
+            ]},
+            {"canShineCharge": {"usedTiles": 13, "gentleDownTiles": 2, "gentleUpTiles": 2, "openEnd": 0}}
+          ]},
+          {"and": [
+            {"doorUnlockedAtNode": 2},
+            {"doorUnlockedAtNode": 3},
+            {"canShineCharge": {"usedTiles": 14, "gentleDownTiles": 2, "gentleUpTiles": 2, "openEnd": 0}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {}},
+        "canRModeSparkInterrupt"
+      ],
+      "unlocksDoors": [
+        {"nodeId": 2, "types": ["ammo"], "requires": []},
+        {"nodeId": 3, "types": ["ammo"], "requires": []}
+      ],
+      "clearsObstacles": ["A"],
+      "flashSuitChecked": true,
+      "blueSuitChecked": true,
+      "note": [
+        "Kill the pirates to get them out of the way before using a Crystal Flash on the floating platform to open the bomb barrier.",
+        "Alternatively, farm them for energy and use a Power Bomb on the floating platform.",
+        "Shinecharge at the bottom runway and then jump into a Viola to interrupt."
```

Comment:
Should have a note for how to charge kill and pb kill.
PBkill = shoot standing pirate then quickly jump and place a PB fairly low and idle while morphed.  repeat 3 times.
Charge = Interupt standing pirate; Stay crouched shooting charged shots.  At some point jump to make wall pirate cross over.